src/markdown/tutorial/part-1/01-orientation.md

```diff
@@ -206,10 +111,24 @@ pnpm i --save-dev concurrently
 git add package.json
 ```
 
+```run:file:patch hidden=true lang=js cwd=super-rentals filename=app/deprecation-workflow.js
+@@ -22,2 +22,9 @@ setupDeprecationWorkflow({
+     /* { handler: 'silence', matchId: 'template-action' }, */
++    /**
++     * This is to account for a deprecation that shipped in ember-cli 6.4
++     * with ember-data v5.6 which needs a blueprint update to avoid the
++     * deprecation that is otherwise irrelevant for tutorial purposes.
```

Comment:
The deprecation for which I wrote this comment is not there in ember-data 5.7, which is in the classic blueprint for the beta, is it not for the new blueprint?

Reply:
I just meant the deprecation I had been suppressing no long shows up in 5.7 afaik

Reply:
@kategengler I updated the comment to just be a simple TODO item to remove these deprecations so I'm resolving this conversation 👍